make/compiler/Makefile.gnu

```diff
@@ -90,12 +90,24 @@ endif
 #
 # query gcc version
 #
+ifndef GNU_GCC_MAJOR_VERSION
+export GNU_GCC_MAJOR_VERSION = $(shell $(CC) -dumpversion | awk '{split($$1,a,"."); printf("%s", a[1]);}')
```

Comment:
Just style, not functionality: Does `cut -d. -f1` not do the job of the awk here? Perhaps 'cut' isn't available everywhere we need it?  (and more of these below)